### ISSUE TITLE
Save button not active when selecting an appointment.

### ISSUE THREAD

**Issue by gutiersa** [openemr/openemr#5043]
### Describe the bug
"Save" button is grayed out (not available) when making a new patient appointment. 

If appointment slot is not available, when attempting to make a new patient appointment, and the dialog window "Provider not available, use it anyway? OK, Cancel" (see below)

![appt0](https://user-images.githubusercontent.com/11306638/160255312-7249efd8-e2be-4661-8174-8cb03b4de971.png)

Then, once I have selected the appropriate, available appointment, I cannot save it.
The subsequent dialog window has an unavailable save button.

### To Reproduce

1. Add an appointment in calendar, but set the time to one that is outside of office hours, say 9 pm. This will allow for the "provider not available" dialogue.
2. On the "Provider not available, use it anyway?" select "cancel"
3. The "Available appointments calendar" dialogue window opens up.
4. Select an available appointment.
5. The "save" button does not work!

### Expected behavior
Save button should  become active once an appointment is selected.

### Client configuration

- Browser: firefox
- OpenEMR version: demo
- Operating system: windows 10

**Comment by stephenwaite** [comment]
> On the "Provider not available, use it anyway?" select "cancel"

hi @gutiersa , at this point, after you click "Cancel", you have to cancel out altogether and start anew, it is a bug but can be worked around

**Comment by infeg-khaleela** [comment]
Hi Team
We have a fix for this issue and would like to contribute. 
please let us know.

Thanks
INFEG TEAM

**Comment by stephenwaite** [comment]
hi @infeg-khaleela , that sounds great, did you check out the [contributing guide](https://github.com/openemr/openemr/blob/master/CONTRIBUTING.md)?

**Comment by gutiersa** [comment]
Yay, luv it

**Comment by infeg-khaleela** [comment]
> hi @infeg-khaleela , that sounds great, did you check out the [contributing guide](https://github.com/openemr/openemr/blob/master/CONTRIBUTING.md)?

Yes we have seen them - would like to become active contributors of the community.

**Comment by infeg-khaleela** [comment]
Here is the pull request for your review.
https://github.com/openemr/openemr/pull/5111

please let us know for any changes if required.

Thank You
INFEG TEAM

**Comment by gutiersa** [comment]
Yay, thank you much.
